src/components/ThankYouResponse.tsx

```diff
@@ -0,0 +1,16 @@
+import { h } from 'preact';
+
+type Props = {
+  title: string;
+  subtext: string;
+};
+
+export const ThankYouResponse = ({ title, subtext }: Props) => {
+  return (
```

Comment:
@afolasope You need to consider that questions can be customised to match the brand. For v1, users should be able to update the colour for `question`, `answer`, `button`, `background`, and `progressBar`.